### PR TITLE
[PR Test] Drag-and-Drop with original Dockerfile

### PR DESCRIPTION
**Pull request by Ojeechan**
- with original Dockerfile of develop branch 
- other files is left as is as the original pull request
- has drag and drop function

### CODE REVIEW

**Comment by Ojeechan** [comment]
@nirlipo @YIDING-CODER 

There might be some misunderstanding due to lack of my knowledge, but I made some PRs to be tested.

Thank you.

**Comment by Ojeechan** [comment]
@nirlipo @YIDING-CODER
Even if I clone the develop branch as it is, and just follow the instruction in REAMDE.md:

> export IMAGE_TAG=planimation-frontend:latest
 export UNITY_LICENSE=LICENSE_FILE=Unity_lic.ulf
 docker build --no-cache -t ${IMAGE_TAG} --build-arg LICENSE_FILE=${LICENSE_FILE} -f docker/release/Dockerfile .
 docker run -d -p 8080:80 ${IMAGE_TAG}
 docker ps 

the process of **docker build** seems to fail in building WebGL project inside container.
Is the instruction correct?
I'd just like to know why Travis.CI passes a test but would fail in my host environment.

**Comment by nirlipo** [comment]
Hi @Ojeechan, Thanks for doing the tests. It seems that with the original docker file Travis works now.

With respect to running the same command locally, How did you decode the Unity_lic.ulf.enc file? I'm trying to replicate your error in my machine but can't get the encryption key from Travis.

**Comment by Ojeechan** [comment]
Hi @nirlipo

I actually did not decode the Unity_lic.ulf.enc, but just logged in to the frontend cotainer and fetched a copy of Unity_lic.ulf file to my host OS.

Thank you.

**Comment by Ojeechan** [comment]
@nirlipo 
I'm sorry, 

> I actually did not decode the Unity_lic.ulf.enc, but just logged in to the frontend cotainer and fetched a copy of Unity_lic.ulf file to my host OS.

I feel this seems not to be what I actually did.
Please let me check my memory...

**Comment by Ojeechan** [comment]
@nirlipo 
I totally forgot that the Unity_lic.ulf file I used is not the decrypted license, but just the one extracted from Unity project on host OS.
I put it as a stub and proceeded to docker build process.

Now it is clear that my question (https://github.com/planimation/frontend/pull/56#issuecomment-633177447) is irrelevant with Travis problem.

By the way, can I ask you the way how to decrypt the license file Unity_lic.ulf.enc locally?
Is it available for me to do just in the same way Travis does, like using an AES key?

Thank you.

**Comment by nirlipo** [comment]
Thanks @Ojeechan , In order to decript the Unity_lic.ulf file we need to access the environment variables from Travis. The only issue is that I didn't set-up Travis, so I can't retrieve the passcode that was used. I tried to reach out to @rakeshchoyal92 by email. If he doesn't get back to us, we may have to setup the env variables again with the a new licence if we want to replicate the command locally.

In the meantime I'll merge this PR with develop, as it passes all tests

**Comment by Ojeechan** [comment]
@nirlipo 
Oh!
Thank you very much for taking time!

**Comment by nirlipo** [comment]
Hi @Ojeechan ,

I tested https://planimation.planning.domains/ and something seems broken with the communication with the backend. Can you look at it? We'll have to revert the merge.

**Comment by Ojeechan** [comment]
@nirlipo 
Yes, I have an exact idea about that. 
As mentioned in (https://github.com/planimation/frontend/pull/52#issue-418904513), I left the server as http://127.0.0.1:8000/upload/pddl in SceneCoordinator.cs.
What url should I designate here?

File: Assets/Visualiser/Scripts/ScenesCoordinator.cs [hunk 1]
```diff
@@ -79,10 +79,13 @@ public void planimationPlugin()
         formData.Add(new MultipartFormDataSection("url", customSolverDomain));
         //serialize form fields into byte[] => requires a bounday to put in between fields
         byte[] formSections = UnityWebRequest.SerializeFormSections(formData, boundary);
-        //UnityWebRequest www = UnityWebRequest.Post("http://127.0.0.1:8000/upload/pddl", formData);
+        /* upload test for drag and drop function */
+        UnityWebRequest www = UnityWebRequest.Post("http://127.0.0.1:8000/upload/pddl", formData);
 		//UnityWebRequest www = UnityWebRequest.Post("https://planning-visualisation-solver.herokuapp.com/upload/pddl", formData);
-        UnityWebRequest www = UnityWebRequest.Post("/upload/pddl", formData);
```
Comment (nirlipo) [inline Assets/Visualiser/Scripts/ScenesCoordinator.cs:84]: @Ojeechan ,revert back to this URL, as it uses https://planimation.planning.domains/upload/pddl

Comment (Ojeechan) [reply Assets/Visualiser/Scripts/ScenesCoordinator.cs:84]: I will do that.
Thank you very much!

**Pull request merged**